py/error_handler.py

```diff
@@ -0,0 +1,41 @@
+class ErrorHandler( object ):
+    
+    def __init__(self, status_code):
+        self.status_code = status_code
+        self.error_codes = {
+            # "201":{
+            #     "/api/create/":["Created", "application/json (object or array of objects) created object(s)"]
+            # },
+            # "207":{
+            #     "/api/create/":["", "application/json (object or array of objects) created object(s)"]
+            # },
+            "400":{
+                "/api/create/":["Bad Request", "application/json (object or array of objects) created object(s)"],
```

Comment:
Why not use the server's response as text?

Reply:
https://github.com/remyla/damas-core/issues/244
It mentioned here to follow the link and error code given for response, can you please verify the same.We can send server response if you verify.

Reply:
Requirement given on #244:

"As an example, the create function returns a JSON or None but could return the error code to precisely return the cause of the failure. The exact specification and the different errors codes returned by the server are described here.

This can be achieved using an error object which will be returned as an argument to the async method (which needs to be done). When we use a sync method, we need to use the most relevant way: redo the methods so that we return an object containing the return value plus the error? This way, we would break the compatibility with the current API."

Reply:
yes the link is pointing to the specifications of the server, containing the error codes, and the possible values returned (no need to recreate messages on the client side since the server is sending it)

Reply:
ok